modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/engine/ObjectActionEngineImpl.java

```diff
@@ -195,9 +189,14 @@ private void _executeObjectActions(
 					objectDefinition.getObjectDefinitionId(),
 					objectActionTriggerKey)) {
 
-			_executeObjectAction(
-				objectAction, objectDefinition, payloadJSONObject, userId,
-				variables);
+			try {
+				_executeObjectAction(
+					objectAction, objectDefinition, payloadJSONObject, userId,
+					variables);
+			}
+			catch (Exception exception) {
```

Comment:
In this commit, the intention is to throw the exception.
Why do we need this try/catch in this method?

Reply:
I decided to keep the old behavior for actions that are not `standalone` since we don't need to throw exceptions for the other actions.